libbeat/docs/shared-docker.asciidoc

```diff
@@ -76,12 +76,12 @@ docker run \
   setup -E setup.kibana.host=kibana:5601 \
   -E output.elasticsearch.hosts=["elasticsearch:9200"] <1> <2>
 --------------------------------------------
-endif::[]
 
 <1> Substitute your Kibana and Elasticsearch hosts and ports.
 <2> If you are using the hosted Elasticsearch Service in Elastic Cloud, replace
 the `-E output.elasticsearch.hosts` line with the Cloud ID and elastic password
 using this syntax:
+endif::[]
```

Comment:
This will result in errors when we build the Beats docs. The callouts are valid for every example in sequence of conditional blocks, so what you need to do is add APM to one of the conditional statement, or add a new example for APM. (The same applies below.)

@DanRoscigno recently updated the Beats docs, but APM wasn't on his radar (or mine) because I didn't realiE you were reusing this file.